tests/cycles.rs

```diff
@@ -48,9 +48,6 @@ use salsa::Durability;
 // | Cross  | Fallback | N/A      | Tracked   | both     | parallel/parallel_cycle_mid_recover.rs |
 // | Cross  | Fallback | N/A      | Tracked   | both     | parallel/parallel_cycle_all_recover.rs |
 
-// TODO: The following test is not yet ported.
-// | Intra  | Fallback | Old      | Tracked   | direct   | cycle_disappears_durability |
-
```

Comment:
Did you remvoe the TODO because the test is now ported?

Reply:
Yes; the test exists in the file and isn't commented out and runs and passes, so it doesn't seem like anything more is needed.